sources/logdog/conf/logdog.vmware-dev.conf

```diff
@@ -0,0 +1,2 @@
+exec docker-info docker info
```

Comment:
I think we could have this be a link to the aws-dev file to reflect that we expect no differences.  (We don't expect any, right?)

Reply:
I'll symlink it for now.  If we find differences later, we'll rectify it then.